homeassistant/components/olarm/strings.json

```
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "step": {
+      "device": {
+        "title": "Select Device",
+        "description": "Choose the Olarm device you would like to integrate. If your device is not listed, please ensure you have enabled public API access in your Olarm account for that device.",
+        "data": {
+          "select_device": "Device",
+          "load_zones_bypass": "Would you like to also load entities for zone bypassing?"
+        },
+        "data_description": {
+          "select_device": "",
+          "load_zones_bypass": ""
+        }
+      }
+    },
+    "error": {
+      "invalid_auth": "Invalid authentication, please try again.",
+      "oauth_failed": "OAuth authentication failed, please try again."
+    },
+    "abort": {
+      "already_configured": "Account is already configured",
+      "already_in_progress": "Configuration flow is already in progress",
```

There are common strings for those which save translator time to do them again:

```suggestion
      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
      "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
```

Perhaps you also find some that match some of those below.